src/Console/Command/FixCommand.php

```diff
@@ -283,7 +283,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if ('none' === $progressType || null === $stdErr) {
             $progressOutput = new NullOutput();
         } else {
-            $finder = new \ArrayIterator(iterator_to_array($finder));
+            $finder = new \ArrayIterator($finder);
```

Comment:
Can we move this line before this `if`? This would remove the need to do the same at line 316.

Reply:
Actually it might be better to replace `count()` with `iterator_count()` so we don't need to rely on `ArrayIterator`.